schema.graphql

```diff
@@ -117,4 +119,21 @@ type TraderDayData @entity {
   fee: BigInt! # trading fee aggregated per day for trader
   fundingPayment: BigInt! # funding payment aggregated per day for trader
   realizedPnl: BigInt! # total pnl aggregated per day for trader
+}
+
+type ReferralCode @entity {
+  id: ID!
+  referrer: Trader! # referrer address
+  referees: [Bytes!]! # referees
+  createdAt: BigInt! # created at
```

Comment:
can we also have a reverse lookup for `ReferralCodeDayData` as well? Similar to `Trader.dayData`